app/controllers/cards/wizards_controller.rb

```diff
@@ -4,7 +4,8 @@ class Cards::WizardsController < ApplicationController
   layout 'admin'
 
   def new
-    @card = current_user.cards.find_by(status: nil)
+    # first match (rewhere is need because of default scope)
+    @card = Card.rewhere(status: :incomplete, user: current_user).first
```

Comment:
I would find more intuitive to have a `with_incomplete` scope than using a rewhere.